README.md

```diff
@@ -50,7 +61,7 @@ See [additional configuration options](#additional-apollo-config-options).
 
 <h3 id="api-key">Setting up the <code>.env</code> file</h3>
 
-To authenticate with GraphOS Studio to pull down your schema, create a `.env` file in the same directory as the `apollo.config.js` file. This should be an untracked file (that is, don't commit it to Git).
+To authenticate with GraphOS Studio to pull down your schema, create a `.env` file in the same directory as the `apollo.config.json` file. This should be an untracked file (that is, don't commit it to Git).
 
 Then go to your [User Settings page](https://studio.apollographql.com/user-settings/api-keys?referrer=docs-content) in GraphOS Studio to create a new Personal API key.
```

Comment:
Is this different than the API key they needed to obtain in Line 44?

Reply:
No, the exact same one. This whole thing is just a bit too repetitive :sweat_smile: